CHANGES.md

```diff
@@ -1,6 +1,11 @@
 # tulip changelog
 
 
+## 1.3.1
```

Comment:
Should this be 1.4.0 ? Removal of the `jtlv` interface is not a backwards-compatible change.

There are several issues marked for [the version 1.3.1 milestone](https://github.com/tulip-control/tulip-control/milestone/8), but they can just as well be included in version 1.4.0.